test/suite/test_util01.py

```diff
@@ -141,7 +145,24 @@ def dump_kv_to_line(self, b):
         # The output from dump is a 'u' format.
         return b.strip(b'\x00').decode() + '\n'
 
-    def dump(self, usingapi, hexoutput):
+    def write_entries(self, cursor, expectout, hexoutput, commit_timestamp, write_expected):
+        if commit_timestamp:
+            self.session.begin_transaction()
+        for i in range(0, self.nentries):
+            key = self.get_key(i)
+            value = 0
+            if not write_expected:
```

Comment:
we could reverse the if conditions to make it read a bit easier:

```
if write_expected:
    value = self.get_value(i)
else:
    value = self.get_value(i + random.randint(1, self.nentries))
```

Not a strong opinion on this one, so don't mind if you keep as such.

Reply:
done!